.github/workflows/dependabot-pr-check-yml

```diff
@@ -0,0 +1,12 @@
+### .github/workflows/dependabot-pr-check.yml
+### This workflow doesn't have access to secrets and has a read-only token
```

Comment:
Can we add a link to the appropriate GitHub issue / comment talking about why this is needed here?

Reply:
@merlinc This has been added and to the PR description as well